build/darwin/create-universal-app.ts

```diff
@@ -87,8 +87,9 @@ async function main() {
 	});
 	await fs.writeJson(productJsonPath, productJson);
 
-	// Verify if native module architecture is correct
-	const findOutput = await spawn('find', [outAppPath, '-name', 'keytar.node']);
+	// {{SQL CARBON EDIT}}
+	// Verify if native module architecture is correct (lookup only in unpacked node_modules)
```

Comment:
Let's make it clearer what part of the comment is from our edit (so people looking through the comments are quickly able to tell what came from VS Code and what came from our changes)

```suggestion
	// Verify if native module architecture is correct
	// {{SQL CARBON EDIT}} Some of our extensions have their own keytar so lookup 
	//   only in core modules since this code doesn't work with multiple found modules. 
	//   We're assuming here the intent is just to check a single file for validation and not
	//   needing to check any others since this currently is ignoring all other native modules. 
```